Require stable version of DoctrineModule 1.2.0 instead of dev-master

webimpress opened the pull request:
Thanks @gianarb for DoctrineModule 1.2.0 release with ZF3 compatibility ! 👍

gianarb commented:
I am not sure about this.. I think that we can work with `>=1.0` or something similar.. 
What do you think @webimpress ?

webimpress commented:
@gianarb Probably we can. Depends on policy you have in Doctrine. In Zend modules we updated all packages to require last stable version. If somebody can update this library to the latest version it means his code is also compatible with DoctrineModule 1.2.0 and it should be also updated IMO.

gianarb commented:
Right :) Thanks
